src/internal/packager/helm/repo.go

```diff
@@ -165,3 +166,18 @@ func (h *Helm) DownloadChartFromGitToTemp(spinner *message.Spinner) (string, err
 
 	return gitCfg.GitPath, nil
 }
+
+// Helper function src/pkg/packager/prepare.goL118 src/pkg/packager/create.goL328
+func (h *Helm) TransformGitURL(charts string) (string, error) {
```

Comment:
Can we rename this function? Since it's calling `h.PackageChartFromGit(...)` it is doing a lot more than just transforming the git URL.

Reply:
https://github.com/defenseunicorns/zarf/pull/1892 (We deleted this function). Thanks for the feedback!